docs/README.md

```diff
@@ -93,5 +93,22 @@ sudo cp -r * /var/www/html/
 
 You can then access the html files at `http://localhost/index.html`.
 
+## Updating Commands Reference topic
+
+Updating content in the [Commands Reference](https://hyperledger-fabric.readthedocs.io/en/latest/command_ref.html) topic requires additional steps. Because the information in the Commands Reference topic is generated content, you cannot simply update the associated markdown files. Instead you need to update the associated  `_preamble.md` or `_postscript.md` files for the command which resides under `src/github.com/hyperledger/fabric/docs/wrappers`. Then you need to run the command `make help-docs` which generates the updated markdown files under `docs/source/commands`. Remember that when you push the changes to GitHub, you  need to include the `_preamble.md` or `_postscript.md` file that was modified as well as the generated markdown file.
+
+**Tip:** Before running `make help-docs`, ensure that your Go library is at 1.11.5 or higher.
```

Comment:
Instead of adding another prereq version that will need to be updated, I'd suggest just reference the main Go prereq here:
https://hyperledger-fabric.readthedocs.io/en/latest/prereqs.html#go-programming-language